src/app/modules/shared/components/details-fields/details-fields.component.ts

```diff
@@ -246,6 +246,12 @@ export class DetailsFieldsComponent implements OnChanges, OnInit {
     this.saveEntry.emit({ entry, shouldRestartEntry: this.shouldRestartEntry });
   }
 
+  onclickFormAction(isProjectSelected: boolean){
+    if (isProjectSelected){
+      this.toastrService.warning('Please, select first a project');
```

Comment:
I think we could change the message to "Please, first select a project"

Reply:
Ok i´m going to  change it